Sources/Sentry/Public/SentryOptions.h

```diff
@@ -462,6 +462,13 @@ NS_SWIFT_NAME(Options)
 
 #endif
 
+/**
+ * Use this in conjunction with `enableUIViewControllerTracing`.
+ * By enabling this, every UIViewController tracing transaction will wait
+ * for a call to `SentrySDK.reportFullDisplay()` until the transaction times out.
```

Comment:
`m`: I think we have to be more specific here. What happens when you don't call `SentrySDK.reportFullDisplay()`? Does the SDK throw the transaction away after the timeout?